scripts/airdrop.js

```diff
@@ -0,0 +1,54 @@
+const fs = require("fs").promises
+
+const { signAndSend, transactionExistsOnSafeServer } = require("./utils/gnosis_safe_server_interactions")(web3, artifacts)
+const { buildTransferDataFromList } = require("./utils/trading_strategy_helpers")(web3, artifacts)
+const { promptUser } = require("./utils/user_interface_helpers")
+const { default_yargs } = require("./utils/default_yargs")
+const argv = default_yargs
+  .option("masterSafe", {
+    type: "string",
+    describe: "Address of Gnosis Safe transfering funds",
+    demandOption: true,
+  })
+  .option("transferFile", {
+    type: "string",
+    describe: "file name (and path) to the list transfers",
+    demandOption: true,
+  })
+  .option("verify", {
+    type: "boolean",
+    default: false,
+    describe: "Do not actually send transactions, just simulate their submission",
+  })
+  .option("useWei", {
+    type: "boolean",
+    describe: "boolen flag used to indicate if the amounts provided are already in atoms",
```

Comment:
typo in `boolen`, actually. The type says already boolean. I think we can remove the type from the description "Flag used to ..."

Reply:
I have removed useWEI as it is too much unnecessary maintenance.